### DISCUSSION TITLE
Exposing 3rd-party C++ classes and functions

### DISCUSSION THREAD

**Discussion by utilitydelta** [nxrighthere/UnrealCLR#234] [General]
Hi,
We were thinking about adding support for the new SDMC mesh component (see http://www.gradientspace.com/tutorials/2021/01/19/the-interactive-tools-framework-in-ue426) but noticed that its not so easy... 

Any instructions anywhere on how to do this? Is the code generator you use to update Codegen.cs published anywhere? How does it work?

Thanks!
Tyson

**Comment by nxrighthere** [comment]
Please, see https://github.com/nxrighthere/UnrealCLR/discussions/161 and https://github.com/nxrighthere/UnrealCLR/discussions/56#discussioncomment-417105.

**Reply by utilitydelta** [reply]
ah, thanks for that :)
